packages/hardhat-helpers/src/localhost.js

```diff
@@ -0,0 +1,19 @@
+export const isLocalhost = async () => {
+  const { ethers } = require('hardhat')
+  const { chainId } = await ethers.provider.getNetwork()
+  return chainId.toString() === '31137'
+}
+
+export const getEvent = async (tx, eventName) => {
```

Comment:
Is this the right place for this helper?

Reply:
yes true, moved to its own file